docs/cluster-management/configure-api.md

```diff
@@ -226,6 +226,17 @@ notifications:
 
 Configurable authentication settings have not yet been built, but can easily be added. We’re using the [nodemailer](https://nodemailer.com/about/) package to power emails, so authentication features will be similar to any typical nodemailer setup. Contribute at: [screwdriver-cd/notifications-email](https://github.com/screwdriver-cd/notifications-email)
 
+#### Slack Notifications
+
+Create a `screwdriver-bot` user in your Slack instance. Generate a Slack token for the new user and set it in your Slack notifications settings.
```

Comment:
`screwdriver-bot` is a bot user not a regular user. Can we say `screwdriver-bot` bot user and link it with https://api.slack.com/bot-users?